nsv13/code/controllers/subsystem/overmap_mode.dm

```diff
@@ -597,20 +646,15 @@ SUBSYSTEM_DEF(overmap_mode)
 			var/amount = input("Enter amount of threat to add (or substract if negative)", "Adjust Threat") as num|null
 			SSovermap_mode.modify_threat_elevation(amount)
 		if("change_gamemode")
-			if(SSovermap_mode.mode_initialised)
-				message_admins("Post Initilisation Overmap Gamemode Changes Not Currently Supported") //SoonTM
+			if(SSticker.HasRoundStarted())
+				message_admins("Mid-round Overmap Gamemode Changes Not Currently Supported") //SoonTM
```

Comment:
This is weird; `mode_initialized` should only be `TRUE` if the game did already start (because `ssticker` does that on roundstart). And in case it for some reason already initialized before game start, it WILL runtime on game start because `setup_overmap_mode` is called again.